src/cmd/run.js

```diff
@@ -163,7 +164,7 @@ export default async function run(
   {
     sourceDir, artifactsDir, firefox, firefoxProfile,
     preInstall = false, noReload = false,
-    customPrefs,
+    customPrefs, browserConsole = false,
```

Comment:
@aniketkudale do you mind to move the new `browserConsole` property before `customPrefs`?
(and doing the same at line 198)